### DISCUSSION TITLE
Colour of the split border characters

### DISCUSSION THREAD

**Discussion by pezz** [scandum/tintin#50] [General]
Hi,

Pretty new and just getting into tintin configs. I can't see if there is any way in the manual to set the fg/bg colour of the char that's used to draw the split lines?

Just curious.

Cheers.

**Comment by scandum** [comment]
You can try something like:
```
#screen fill default green
#screen fill default <114>
etc
```
Or
```
#draw {red b blue} tubed horizontal line -2 1 -2 -1
```

#help color has some additional information about color codes.

**Reply by pezz** [reply]
Oh nice, lots to play with here.

Thanks for the reply!
